.github/workflows/ci.yml

```diff
@@ -180,9 +180,9 @@ jobs:
           sudo apt install cmake clang-19 llvm g++-12 valgrind zlib1g-dev libcurl4-openssl-dev
           # Install kcov from source
           sudo apt-get install binutils-dev libssl-dev libelf-dev libstdc++-12-dev libdw-dev libiberty-dev
-          git clone https://github.com/SimonKagstrom/kcov.git
+          git clone https://github.com/jpnurmi/kcov.git
           cd kcov
-          git checkout v43 # pin to a version without the current coveralls git integration
+          git checkout v43-opt # pin to a version without the current coveralls git integration
```

Comment:
While I prefer to use pinning here, and no new version with a fix has been tagged or released yet, I already fixed the coveralls git integration upstream: https://github.com/SimonKagstrom/kcov/pull/482